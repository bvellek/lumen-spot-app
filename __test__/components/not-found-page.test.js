import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '../../public/js/components/not-found-page';

describe('<NotFoundPage />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <NotFoundPage />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
