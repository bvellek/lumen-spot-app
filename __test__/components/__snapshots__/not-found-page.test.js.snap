// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> it renders correctly 1`] = `
<div
  className="not-found"
>
  <h1>
    404
  </h1>
  <h2>
    Page not found!
  </h2>
  <p>
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Go back to the main page
    </a>
  </p>
</div>
`;
